### ISSUE TITLE
Support zip/deflate in AIP compression

### ISSUE THREAD

**Issue by helenst** [archivematica/Issues#493]
**Please describe the problem you'd like to be solved.**
Our Archivematica-generated AIPs need to be submitted to a Java-based storage engine which expects bags in the form of a zip archive and does not support any of the formats currently output by Archivematica.

**Describe the solution you'd like to see implemented.**
To add zip / deflate as an available option in the "Compression method" dropdown, include support for that method in the compression script, and include the unzip tool in MCPClient containers so that `atool` can decompress zipped archives.

**Describe alternatives you've considered.**
The alternative was to modify the service to accept .7z files, however this is not possible at the moment.

**Additional context**
I'm working on a PR for this. :sparkles: 

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by ross-spencer** [comment]
Hi @helenst how is this work going? We've still a `triage-1.10` label on this and it looks like I have capacity to CR the work once it is complete. I have a couple of questions:

1. How long do you think it will take to address the storage-service part of this work noted in #681?
2. In the Wellcome [epic](https://github.com/wellcometrust/platform/issues/3535) you note that this feature isn't needed by Wellcome any more. Given that, it might best to focus on #681, and close this ticket and the corresponding PR, over perhaps introducing the maintenance overhead of this other mechanism? It can always be re-opened if another user determines it a necessity? Will that approach be okay? 

**NB.** I've attached the `discussion` label. Other users might already have that need.

**Comment by helenst** [comment]
Hi @ross-spencer I've been working on the storage service part of the work. I think it works, I've been doing a little bit of tidy up work and adding some tests, since there are quite a few points in the code that needed to change! I've also been addressing #605 while I'm there. If you'd rather leave out the zip/deflate work, that's fine as far as I'm concerned. We don't use it now and are using gzip instead. I think zip can be relatively easily added later on if somebody needs it.

**Comment by ross-spencer** [comment]
Cool, thanks @helenst if you leave the zip/deflate work for now, we can always pick it up again in future. Once you've submitted the PR ping me and I'll turn it around as quickly as possible. Very excited for the tests!

**Comment by sromkey** [comment]
Because it's not needed for now I'm going to close this issue but if it ever comes up as being of interest to someone in the future we can re-visit it!
